package-lock.json

```diff
@@ -4554,7 +4539,7 @@
         "glslify": "^6.1.0",
         "robust-orientation": "^1.1.3",
         "split-polygon": "^1.0.0",
-        "vectorize-text": "^3.0.0"
+        "vectorize-text": "git://github.com/archmoj/vectorize-text.git#457db88a269e6412c6f6007f1ba88d06bdfbc08c"
```

Comment:
Link to corresponding PR on `vectorize-text` module
https://github.com/mikolalysenko/vectorize-text/pull/8